romana-install/config.rb.example

```diff
@@ -0,0 +1,5 @@
+#$controller_vm_memory=512
```

Comment:
The example value should probably be the 4GB that devstack needs on the controller.